packages/webpack/example/webpack.config.js

```diff
@@ -0,0 +1,18 @@
+const path = require('path');
+const HoneybadgerSourceMapPlugin = require('@honeybadger-io/webpack')
+
+module.exports = {
+  entry: './src/index.js',
+  mode: 'production',
+  output: {
+    filename: 'main.js',
```

Comment:
🤔 I'd assume this would be named `index.js`, but no big deal.

Reply:
I followed the webpack intro project instructions 🤷

Reply:
But I agree with you 😆